string_commands.go

```diff
@@ -32,37 +32,43 @@ type StringCmdable interface {
 
 func (c cmdable) Append(ctx context.Context, key, value string) *IntCmd {
 	cmd := NewIntCmd(ctx, "append", key, value)
+	cmd.keys = append(cmd.keys, key)
```

Comment:
Why do you need to append key here? You pass key as an argument in constructor `NewIntCmd(ctx, "append", key, value)`, so it's expected that by calling `cmd.Keys()` you'll get back your keys.